azure-pipelines.yml

```diff
@@ -282,44 +259,6 @@ jobs:
       includePatterns:
         - "/hw/top_earlgrey/Vchip_earlgrey_verilator"
 
-# Software targeting the English Breakfast top level is produced by patching
-# the source tree before building. This builds a selected subset of software
-# only.
-# TODO: This is a rather ugly hack, which will go away once we properly support
-# building more than one top-level design with different parametrizations.
-# Work towards this goal is tracked in issue #4669.
-- job: build_and_execute_verilated_tests_englishbreakfast
-  displayName: Verilated English Breakfast (Execute)
-  # Build and execute tests on the Verilated English Breakfast toplevel design with Bazel
-  pool:
-    vmImage: ubuntu-20.04
-  dependsOn: chip_englishbreakfast_verilator
-  steps:
-  - template: ci/checkout-template.yml
-  - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_englishbreakfast_verilator
-  - template: ci/load-bazel-cache-write-creds.yml
-  - bash: |
-      . util/build_consts.sh
-      ci/scripts/run-english-breakfast-verilator-tests.sh
-    displayName: Execute tests
-  - bash: |
-      . util/build_consts.sh
-      mkdir -p "$BIN_DIR/sw/device/lib/testing/test_rom"
-      cp $(ci/scripts/target-location.sh \
-          //sw/device/lib/testing/test_rom:test_rom_fpga_cw305 \
-          --features=-rv32_bitmanip \
-          --copt=-DOT_IS_ENGLISH_BREAKFAST_REDUCED_SUPPORT_FOR_INTERNAL_USE_ONLY_) \
-        "$BIN_DIR/sw/device/lib/testing/test_rom"
-    displayName: Copy test_rom_fpga_cw305 to $BIN_DIR
-  - template: ci/upload-artifacts-template.yml
-    parameters:
-      includePatterns:
-        - "/sw/device/lib/testing/test_rom/test_rom_fpga_cw305.32.vmem"
```

Comment:
What's this for?

Reply:
I'm not sure, I can't find any other reference to it. It's pretty easy to build so I'm not sure we need to upload it here.